maven-failsafe-plugin/pom.xml

```diff
@@ -45,6 +45,13 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>2.0.16</version>
```

Comment:
Please use 1.7.x as Maven 3.x support only this version

Reply:
Changed slf4j-simple version to 1.7.36.   Thank you.